__tests__/app.test.js

```diff
@@ -112,3 +112,45 @@ describe('GET /api/articles/:article_id', () => {
             });
     });    
 });
+
+
+describe('GET /api/articles', () => {
+    test('200: should return an array of article objects containing the expected fields', () => {
+        return request(app)
+            .get('/api/articles')
+            .expect(200)
+            .then(({body}) => {
+                expect(body.articles.length).toBe(data.articleData.length);
+                expect(typeof body.articles[0]).toBe('object');
```

Comment:
This line would also pass is body.articles[0] was an array - you may want to also do Array.isArray is false.